src/javascript/EditPanelContainer/EditPanel/EditPanelContent/PreviewContainer/PreviewContainer.jsx

```diff
@@ -26,11 +25,31 @@ const styles = theme => ({
     }
 });
 
-const PreviewContainerCmp = ({t, classes, formik}) => {
-    const editorContext = useContentEditorContext();
+const Preview = React.memo(({path, lang, workspace}) => {
```

Comment:
Ici tu as 2 composants dans un même fichier

Reply:
Indeed, next commit moves the component to its dedicated file.